Core/src/LongRunning/LROTraitV2.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Core\LongRunning;
+
+use Google\ApiCore\Serializer;
+use Google\Cloud\Core\Iterator\ItemIterator;
+use Google\Cloud\Core\Iterator\PageIterator;
+use Google\Cloud\Core\RequestHandler;
+# TODO: Point to the correct request class
+use Google\LongRunning\ListOperationsRequest;
+
+/**
+ * Provide Long Running Operation support to Google Cloud PHP Clients.
+ *
+ * This trait should be used by a user-facing client which implements LRO.
+ */
+trait LROTraitV2
```

Comment:
We would try to avoid using version numbers in the class/trait names.

Reply:
Hmm, I will find a better name, how about `LROManagerTrait`?

Reply:
`Manager` would be more for a class than a trait. Something like `LROManagementTrait` would make more sense. However, I think the verb is not necessary. What about simply `LongRunningOperationTrait`?

Reply:
`LongRunningOperationTrait` sounds great, will change to that name